tensorflow/python/ops/parallel_for/pfor.py

```diff
@@ -1612,14 +1613,20 @@ def _add_to_stack(x):
           pfor_inputs = _PforInput(self, y_op, converted_inputs)
           try:
             try:
-              new_outputs = converter(pfor_inputs)
+              if (converter.__name__ == "_fallback_converter"):
```

Comment:
A better approach is to do `converter = functools.partial(_fallback_converter, root_cause="...")` in line https://github.com/tensorflow/tensorflow/blob/4c464afd3d48e175d32f16a2d67c4902ba6624fb/tensorflow/python/ops/parallel_for/pfor.py#L1601

And the root cause should be `"has no variant outputs and has no vectorized variant inputs"`.